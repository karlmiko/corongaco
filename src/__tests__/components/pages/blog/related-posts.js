import React from 'react'
import renderer from 'react-test-renderer'
import RelatedPosts from '~components/pages/blog/related-posts'

describe('Components : Pages : Blog : Byline', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <RelatedPosts
          posts={[
            {
              authors: [
                {
                  name: 'Author Name',
                },
              ],
              title: 'A COVID Blog Post',
              slug: 'a-covid-blog-post',
              publishDate: 'May 6, 2020',
            },
            {
              authors: [
                {
                  name: 'Author Middle Last',
                },
              ],
              title: 'Another Blog Post',
              slug: 'another-blog-post',
              publishDate: 'April 6, 2020',
            },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
