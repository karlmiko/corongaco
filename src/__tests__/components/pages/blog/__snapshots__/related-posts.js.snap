// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Byline renders correctly 1`] = `
<div
  className="container"
>
  <h2>
    Related
  </h2>
  <div>
    <h3>
      <a
        href="/blog/a-covid-blog-post"
      >
        A COVID Blog Post
      </a>
    </h3>
    <p
      className="oneLineByline"
    >
      By
       
      <span
        className="author"
      >
        <span
          className="authorInner"
        >
          Author Name
        </span>
      </span>
      <span
        className="relatedDate"
      >
        May 6, 2020
      </span>
    </p>
  </div>
  <div>
    <h3>
      <a
        href="/blog/another-blog-post"
      >
        Another Blog Post
      </a>
    </h3>
    <p
      className="oneLineByline"
    >
      By
       
      <span
        className="author"
      >
        <span
          className="authorInner"
        >
          Author Middle Last
        </span>
      </span>
      <span
        className="relatedDate"
      >
        April 6, 2020
      </span>
    </p>
  </div>
</div>
`;
